modules/replicator/src/main/java/org/apache/ignite/internal/replicator/message/ReplicaRequest.java

```diff
@@ -24,5 +24,7 @@
  */
 public interface ReplicaRequest extends NetworkMessage {
     /** Gets a replication group id. */
+    // TODO https://issues.apache.org/jira/browse/IGNITE-22522
```

Comment:
I still hope that some day, we will refactor CMG/MG to use replicas. In that case we may introduce MGReplciationGroupId or something like that. Thus it seems reasonable to leave ReplicaRequest as is using ReplicationGroupId.